.nvmrc

```diff
@@ -1 +1 @@
-16
+lts/gallium
```

Comment:
I use https://asdf-vm.com/, which needs `.nvmrc` to have a `lts/*` syntax, or the exact version, e.g. `16.0.0`. This change shouldn't impact anyone using `nvm`